---
title: "Adjusting for background mutation frequency biases improves the identification of cancer driver genes."
date: 2013-09-01
publishDate: 2019-12-12T16:26:43.088421Z
authors: ["Perry Evans", "Stefan Avey", "Yong Kong", "Michael Krauthammer"]
publication_types: ["2"]
abstract: "A common goal of tumor sequencing projects is finding genes whose mutations are selected for during tumor development. This is accomplished by choosing genes that have more non-synonymous mutations than expected from an estimated background mutation frequency. While this background frequency is unknown, it can be estimated using both the observed synonymous mutation frequency and the non-synonymous to synonymous mutation ratio. The synonymous mutation frequency can be determined across all genes or in a gene-specific manner. This choice introduces an interesting trade-off. A gene-specific frequency adjusts for an underlying mutation bias, but is difficult to estimate given missing synonymous mutation counts. Using a genome-wide synonymous frequency is more robust, but is less suited for adjusting biases. Studying four evaluation criteria for identifying genes with high non-synonymous mutation burden (reflecting preferential selection of expressed genes, genes with mutations in conserved bases, genes with many protein interactions, and genes that show loss of heterozygosity), we find that the gene-specific synonymous frequency is superior in the gene expression and protein interaction tests. In conclusion, the use of the gene-specific synonymous mutation frequency is well suited for assessing a gene's non-synonymous mutation burden."
featured: false
publication: "*IEEE transactions on nanobioscience*"
tags: ["Computational Biology", "Gene Expression Profiling", "Genes", "Neoplasm", "Humans", "Loss of Heterozygosity", "Melanoma", "Models", "Genetic", "Mutation", "Mutation Rate", "Tumor Cells", "Cultured", "Ultraviolet Rays", "genetics", "metabolism", "methods", "radiation effects"]
doi: "10.1109/TNB.2013.2263391"
---

